site/en/user_guide.md

```diff
@@ -169,6 +169,12 @@ for i in OCR-D-IMG/*.tif; do base=`basename ${i} .tif`;docker run --rm -u $(id -
 
 In the end, your METS file should look like this [example METS](example_mets.md).
 
+Alternatively, `ocrd-import` from [workflow-configuration](#workflow-configuration) is a shell script which does all of the above (and can convert arbitrary image formats) automatically. For usage options, see:
+
+```sh
+ocrd-import -h
```

Comment:
could you please add a basic sample call for ocrd-import to make it useable right away?

Reply:
Done!